pkgs/package-overrides.nix

```diff
@@ -72,10 +72,12 @@ in
         prev.extensions.ast;
 
     blackfire =
-      if lib.versionAtLeast prev.php.version "8.0" then
+      if lib.versionAtLeast prev.php.version "8.3" then
+        throw "php.extensions.blackfire requires PHP version >= 8.0 and < 8.3"
+      else if lib.versionAtLeast prev.php.version "8.1" then
         prev.extensions.blackfire
       else
-        throw "php.extensions.blackfire requires PHP version >= 8.0.";
+        final.callPackage ./blackfire/1.88.1.nix { inherit prev; inherit final; };
```

Comment:
Maybe move it into extensions subdirectory?

Reply:
Sure, done.